apps/site/src/components/pages/hub/block-data-container/sandboxed-block-demo.tsx

```diff
@@ -53,6 +33,10 @@ export const SandboxedBlockDemo: FunctionComponent<SandboxedBlockProps> = ({
   const loadedRef = useRef(false);
   const iframeRef = useRef<HTMLIFrameElement>(null);
 
+  const [serviceUsageIsPermitted, setServiceUsageIsPermitted] = useState<
+    boolean | undefined
+  >(undefined);
```

Comment:
Nit: this is equivalent
```suggestion
  const [serviceUsageIsPermitted, setServiceUsageIsPermitted] =
    useState<boolean>();
```

Reply:
Fair enough, updated